test/parse-result.js

```diff
@@ -220,6 +220,10 @@ describe('Parse result namespace', () => {
       expect(sourceMaps.first().toValue()).to.deep.equal([[1, 2]]);
     });
 
+    it('should have a convenience method for retrieving source map', () => {
+      expect(element.sourceMap.first().toValue()).to.deep.equal([[1, 2]]);
```

Comment:
Actually, since there can be multiple source maps, then we shouldn't encourage not handling all of them and it shouldn't return the first one.